sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -255,7 +255,7 @@ def process(self, direct_row):
               "value": mutation.set_cell.value
           }
           micros = mutation.set_cell.timestamp_micros
-          if micros > -1:
+          if micros >= -1:
```

Comment:
why is this conditional? why isnt the timestamp just passed through?

Reply:
I forgot why it was originally set up like this, but I don't see a reason for it